api/src/main/java/keywhiz/api/CreateGroupRequest.java

```diff
@@ -30,22 +30,29 @@
   @JsonProperty
   public final String description;
 
+  @Nullable
+  @JsonProperty
+  public final String app;
+
   public CreateGroupRequest(@JsonProperty("name") String name,
```

Comment:
Yes, just have one constructor and other use cases will have to pass `null` as the app name to indicate absence.